.github/workflows/translations.yml

```diff
@@ -1,8 +1,21 @@
+name: "Push translation template"
+
 on:
   push:
     branches:
       - master
-name: "Push translation template"
+    paths:
+      - '**.json'
+      - '**.cpp'
+      - 'src/**'
+      - 'data/**'
+      - 'lang/extract_json_strings.py'
+      - 'lang/strip_line_numbers.py'
+      - 'lang/po/**.pot'
```

Comment:
I think this line is unnecessary now, the rest looks okay.

Reply:
thanks, removed it. Now I think that having both `'**.json'` and `'data/**'` may be unnecessary, no? Same for `'**.cpp'` and `'src/**'`

Reply:
`**.cpp`, `**.h` and `**.json` is consistent with `xgettext ... src/*.cpp src/*.h lang/json/*.py` in `lang/update_pot.sh`